wordpress-mobile/AztecEditor-Android
# Issue/369 update heading menu

### Fix
Update heading menu format button in the format toolbar as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/369.

### Test
1. Launch demonstration app with default content.
2. Notice heading menu displays ***H*** icon.
3. Tap ***Heading*** format button.
4. Notice ***Default*** menu item is selected.
5. Move cursor to "Heading 1" text.
6. Notice head menu displays ***H1*** icon.
7. Tap ***Heading*** format button.
8. Notice ***Heading 1*** menu item is selected.
9. Move cursor to "Heading 2" text.
10. Notice head menu displays ***H2*** icon.
11. Tap ***Heading*** format button.
12. Notice ***Heading 2*** menu item is selected.
13. Move cursor to "Heading 3" text.
14. Notice head menu displays ***H3*** icon.
15. Tap ***Heading*** format button.
16. Notice ***Heading 3*** menu item is selected.
17. Move cursor to "Heading 4" text.
18. Notice head menu displays ***H4*** icon.
19. Tap ***Heading*** format button.
20. Notice ***Heading 4*** menu item is selected.
21. Move cursor to "Heading 5" text.
22. Notice head menu displays ***H5*** icon.
23. Tap ***Heading*** format button.
24. Notice ***Heading 5*** menu item is selected.
25. Move cursor to "Heading 6" text.
26. Notice head menu displays ***H6*** icon.
27. Tap ***Heading*** format button.
28. Notice ***Heading 6*** menu item is selected.
